include/cantera/kinetics/InterfaceKinetics.h

```diff
@@ -287,6 +287,8 @@ class InterfaceKinetics : public Kinetics
      */
     int phaseStability(const size_t iphase) const;
 
+    double InterfaceCurrent(int iPhase);
```

Comment:
Given the other declarations in this file, I wonder if this might be preferred:

```suggestion
    double InterfaceCurrent(const size_t iphase);
```

Reply:
I read about the const qualifier and size_t type and agree with this. I have updated the code here and in other areas to account for this.